src/main/resources/templates/rust-analyzer/settings.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
Perhaps you could install vscode rust to see how it fill configuration settings?